File: 01. Programming Fundamentals/02. Progr. Fundamentals - Exams/Programming Fundamentals Retake Exam - 25 April 2018/Python/01. Rage Expenses.py
import math

lostGamesCount = int(input())
headsetPrice = float(input())
mousePrice = float(input())
keyboardPrice = float(input())
displayPrice = float(input())

headsets = math.floor(lostGamesCount / 2)
mouses = math.floor(lostGamesCount / 3)
keyboards = math.floor(lostGamesCount / 3 / 2)
displays = math.floor(keyboards / 2)

expenses = headsets*headsetPrice+mouses*mousePrice+keyboards*keyboardPrice+displays*displayPrice

print(f"Rage expenses: {expenses:.2f} lv.")

File: 01. Programming Fundamentals/02. Progr. Fundamentals - Exams/Programming Fundamentals Retake Exam - 25 April 2018/PHP/01. Rage Expenses.php
<?php
    $lostGamesCount = readline();
    $headsetPrice = readline();
    $mousePrice = readline();
    $keyboardPrice = readline();
    $displayPrice = readline();

    $headsets = floor($lostGamesCount / 2);
    $mouses = floor($lostGamesCount / 3);
    $keyboards = floor($lostGamesCount / 3 / 2);
    $displays = floor($keyboards / 2);

    $expenses = $headsets * $headsetPrice + $mousePrice * $mouses + $keyboards * $keyboardPrice + $displays * $displayPrice;

    echo 'Rage expenses: ',number_format($expenses,2,'.',''),' lv.';
?>